packages/astro/test/fixtures/content-layer/src/content/config.ts

```diff
@@ -99,11 +99,13 @@ const increment = defineCollection({
 				},
 			});
 		},
+		// Example of a loader that returns an async schema function
+		schema: async () => z.object({
+			lastValue: z.number(),
+			lastUpdated: z.date(),
+		}),
```

Comment:
Did you mean to change this (and the other test changes). We can test this isolated in the content-intellisense fixture instead

Reply:
Yeah, I was testing it with this fixture. I'm fine with leaving these in though, as they catch more cases in the content-layer tests too.